docs/troubleshooting.asciidoc

```diff
@@ -79,56 +81,55 @@ you can try to:
 [[tune-output-config]]
 ==== Tune APM Server output parameters for your Elasticsearch cluster
 
-If your Elasticsearch cluster is sized to properly to ingest more data than what you are currently seeing,
-there are options available to tweak in the APM Server to make better use of the cluster:
+If your Elasticsearch cluster is sized properly, but not ingesting the amount of data you expect, you can tweak APM Server options to make better use of the cluster:
 
 * Increase `output.elasticsearch.workers`
 * Make sure that `output.elasticsearch.bulk_max_size` is high, for example 5120. The default of 50 is very conservative.
-  * If you increase `bulk_max_size`, make sure to also increase `queue.mem.size`.
-  A good rule of thumb for `queue.mem.size` should equal `workers` multiplied by `bulk_max_size`.
++
+If you increase `bulk_max_size`, make sure to also increase `queue.mem.size`.
+A good rule of thumb is that `queue.mem.size` should equal `workers` multiplied by `bulk_max_size`.
 
-Please see {ref}/tune-for-indexing-speed.html[Tune for indexing speed],
-particularly `Increase the refresh interval`, `Disable swapping`,
-`Use faster hardware` and `Indexing buffer size`.
+See {ref}/tune-for-indexing-speed.html[Tune for indexing speed],
```

Comment:
Note that I've removed the hard-coded section titles because this sort of info gets stale over time. If we use a general description, it will be applicable even when the section titles change.